extensions.json

```diff
@@ -730,6 +730,12 @@
       "version": "10.2.0",
       "checkout": "v10.2.0"
     },
+    {
+      "id": "vscode.smali",
+      "repository": "https://github.com/ViRb3/vscode-smali",
+      "version": "1.0.0",
```

Comment:
Since we're publishing from the default branch (i.e. no `"checkout"` specified), there is no need to pin a specific version. (I.e. if `1.0.1` comes out, it's fine to let the script publish it without manual intervention.)

```suggestion
```